hera_mc/mc_session.py

```diff
@@ -723,3 +724,178 @@ def get_paper_temps(self, starttime, stoptime=None):
 
         return self._time_filter(PaperTemperatures, 'time', starttime,
                                  stoptime=stoptime)
+
+    def add_ant_metric(self, obsid, ant, pol, metric, val):
+        """
+        Add a new antenna metric to the M&C database.
+
+        Parameters:
+        ------------
+        obsid: long integer
+            observation identification number
+        ant: integer
+            antenna number
```

Comment:
I can add that if we want it. Is there existing code to get it from obsid + antenna number?